Tests/Resources/Fixtures/config/full.xml

```diff
@@ -44,7 +44,10 @@
                 <my_service type="service" service="my_auth_service"/>
             </authentication>
             <cache cache-pool="my_cache_pool" stream-factory="my_other_stream_factory">
-                <config default-ttl="42" respect-cache-headers="false"/>
+                <config default-ttl="42" respect-cache-headers="false">
+                    <respect-response-cache-directives>X-Foo</respect-response-cache-directives>
```

Comment:
this is supposed to raise an error: https://github.com/php-http/cache-plugin/blob/b2f15cee4ff46f61f7b90137b41627d17fd57be9/src/CachePlugin.php#L69-L72

we should remove the deprecated configuration option from the test. if you want to be precise, you can add a separate legacy test for it